src/chrome/content/rules/BR.xml

```diff
@@ -1,23 +1,42 @@
+<!--
+	No valid Cert for:
+	bayern3-liveblog.br.de
+	liveblog.br.de
+	mediathek.daserste.de
+-->
+
 <ruleset name="Bayrischer Rundfunk">
 
 	<target host="br.de"/>
+	<target host="www.br.de"/>
+	<target host="blog.br.de"/>
 	<target host="cdn-storage.br.de" />
+	<target host="franken-tatort.br.de"/>
+	<target host="frequenzsuche.br.de"/>
+	<target host="jazzwoche-burghausen.br.de"/>
+	<target host="jobs.br.de"/>
+	<target host="mannfrau.br.de"/>
+	<target host="medienundschule.br.de"/>
+	<target host="schnee-von-morgen.br.de"/>
+	<target host="schulkonzert.br.de"/>
 	<target host="story.br.de"/>
-	<target host="www.br.de"/>
+	<target host="tippspiel.br.de"/>
+	<target host="web.br.de"/>
+	<target host="wettermelder.br.de"/>
+	<target host="zuendfunk-netzteil.br.de"/>
+	
+	<target host="brreisen.de"/>
 	<target host="www.brreisen.de"/>
 	<target host="br-ticket.de" />
 	<target host="www.br-ticket.de" />
 	<target host="shop.br-ticket.de" />
 	<target host="www.shop.br-ticket.de" />
-
-	<rule from="^http://(www\.)?shop\.br-ticket\.de/"
-		to="https://br-ticket.muenchenticket.net/" />
-	
-	
-	
-	
 	
+	<target host="www.daserste.de"/>
```

Comment:
`daserste.de`
You should move them to a separate file.

Reply:
Done.